staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5205,6 +5205,9 @@ const (
 	// Local ephemeral storage, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	// The resource name for ResourceEphemeralStorage is alpha and it can change across releases.
 	ResourceEphemeralStorage ResourceName = "ephemeral-storage"
+	// Swap, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
+	// Supported if the feature gate NodeSwap is enabled
+	ResourceSwap ResourceName = "swap"
```

Comment:
I wouldn't expect this here if `swap` isn't being exposed in pod specs

Reply:
After I moved it to node package, there will be a cycle import as the function Swap() in  resource.go.
So I have to move the function to node package as well. Hence, the `func (rl *ResourceList) Swap()` should be something like `func Swap(rl corev1.ResourcesList)`